image/config.go

```diff
@@ -34,38 +32,34 @@ import (
 type config v1.Image
 
 func findConfig(w walker, d *descriptor) (*config, error) {
-	var c config
-	cpath := filepath.Join("blobs", d.algo(), d.hash())
+	var (
+		c     config
+		cpath = filepath.Join("blobs", d.algo(), d.hash())
+	)
```

Comment:
@wking Do you only mean `cpath` is redundant because it is used once? I fixed it.